src/js/routes.js

```diff
@@ -11,123 +10,164 @@ import NotFoundPage from '../pages/404.jsx';
+        component: ActivityForm
+    },
+    {
+        path: '/company',
+        component: CompanyPage
+    },
+    {
+        path: '/about/',
+        component: AboutPage
+    },
+    {
+        path: '/form/',
+        component: FormPage
+    },
+    {
+        path: '/nettverk/',
+        component: NetworkPage
+    },
+    {
+        path: '/logginn/',
+        component: LoginPage
+    },
+    {
+        path: '/registrering/',
+        component: RegisterPage
+    },
+    {
+        path: '/profile/:id',
+        component: Profile
+    },
+    {
+        path: '/profile/',
+        component: Profile
+    },
+    {
+        path: '/dynamic-route/blog/:blogId/post/:postId/',
+        component: DynamicRoutePage
+    },
+    {
+        path: '/conversations/',
+        component: ConversationsPage
+    },
+    {
+        path: '/activities/',
+        component: ActivitiesPage
+    },
+    {
+        path: '/conversations/:chatId',
+        component: ChatPage
+    },
+    {
+        path: '/request-and-load/user/:userId/',
+        async: function(routeTo, routeFrom, resolve, reject) {
+            // Router instance
+            var router = this;
+
+            // App instance
+            var app = router.app;
 
-      // App instance
-      var app = router.app;
+            // Show Preloader
+            app.preloader.show();
 
-      // Show Preloader
-      app.preloader.show();
+            // User ID from request
+            let id = routeTo.params.userId;
 
-      // User ID from request
-      var userId = routeTo.params.userId;
+            fetch(`http://localhost/api/users/${id}`, {headers:{ authorization:localStorage.jwt}})
+                .then(res => {
+                    return res.json();
+                })
+                .then(user => {
+                    // Hide Preloader
+                    app.preloader.hide();
 
-      // Simulate Ajax Request
-      setTimeout(function () {
-        // We got user data from request
-        var user = {
-          firstName: 'Vladimir',
-          lastName: 'Kharlampidi',
-          about: 'Hello, i am creator of Framework7! Hope you like it!',
-          links: [
-            {
-              title: 'Framework7 Website',
-              url: 'http://framework7.io',
-            },
-            {
-              title: 'Framework7 Forum',
-              url: 'http://forum.framework7.io',
-            },
-          ]
-        };
-        // Hide Preloader
-        app.preloader.hide();
+                    // Resolve route to load page
+                    resolve(
+                        {
+                            component: Profile
+                        },
+                        {
+                            context: {
+                                user: user
+                            }
+                        }
+                    );
+                });
+        }
+    },
+    {
+        path: '/request-and-load/user/:userId/',
```

Comment:
Bare fjern hele denne placeholder-routen :)

Reply:
Bro, tror du gjorde noe rart her da du fjernet den routen. nå har du to stykker som går til `'/request-and-load/user/:userId/'` ;)